docs/domain/timeseries/ml-primer/index.md

```diff
@@ -0,0 +1,34 @@
+(timeseries-ml-primer)=
+(tsml-primer)=
+
+# Machine Learning for Time Series Data
+
+A primer about how to apply machine learning procedures to time series data.
+
+
+## Time Series Modeling
+
+This section will introduce you to the concept of time series modeling, and
+discusses the main obstacles usually faced with during its implementation in
+production.
+
+```{toctree}
+:maxdepth: 2
+
+introduction
+anomaly-detection
+```
+
+
+## MLOps
+
+MLOps / ML Ops is a paradigm that aims to deploy and maintain machine learning
+models in production reliably and efficiently, including experiment tracking.
+
+```{toctree}
+:maxdepth: 2
+
+mlops-intro
+mlops-cratedb-mlflow
+mlops-cratedb-sql
+```
```

Comment:
## RFC
Let's discuss the guidance into this content section. Let us know what you think about it, and any other ideas which come to mind when thinking about reshaping and improving this whole content section further. Thanks!

## Status Quo
As you can see, we've already separated the whole large beast into two parts, _**Introduction + Time Series Modeling**_ vs. _**MLOps**_, in order to separate concerns.

So far, we haven't added any _guiding style elements_, like the tiles seen on other pages, yet. Shaping the structure and layout before adding priming and color makes sense.

Another step of the refactoring process so far, already implemented on behalf of 41112ddf17, was to also decrease total page lengths, by breaking out all the hands-on, code-focused sections into dedicated pages. [^1]

[^1]: On the matter of further improvements, please also see https://github.com/crate/cratedb-examples/issues/341#issuecomment-1971228147.